### ISSUE TITLE
[Bug] AltSnap behaves as if the window started moving despite drag threshold was not reached

### ISSUE THREAD

**Issue by emvaized** [RamonUnch/AltSnap#575]
I would like AltSnap to apply some threshold before starting to move the window on left click, so that I can set <kbd>Ctrl</kbd> key as a modifier, but continue using <kbd>Ctrl</kbd>+Click in various apps. 

There is `DragThreshold` option in `AltSnap.ini` config file, which postpones window movement until mouse cursor has moved for some threshold. However, with this option enabled (either `2` or `3`) AltSnap on mouse down still behaves as if the window is already started moving — cursor immediately changes to hand, window opacity changes, and `EVENT_SYSTEM_MOVESIZESTART` is being sent (which I listen for in my Autohotkey scripts). It makes using <kbd>Ctrl</kbd>+Click in apps not very pleasant (although it still works due to `PiercingClick` option being enabled). 

It would be great if AltSnap only started applying all "window started moving" logic only after move threshold was reached, or if there was a separate config option for such a behavior. It also would be great to have a config to change drag threshold, as I couldn't find any in the config file (but that's probably a topic for another issue ticket).

**Comment by RamonUnch** [comment]
Could you try with this version of hooks.dll (x64) replacing the old version.
It should only send the events when movements actually starts.

Also you can change the 

[hooks.zip](https://github.com/user-attachments/files/17925418/hooks.zip)

You also got the option with the above:
```
in the [Advanced] section of the .ini file

NotifyWinEvent=1
; Determines if AltSnap should send the EVENT_SYSTEM_MOVESIZESTART/END
; events when moving a window. This is useful when interacting with
; some other accessibility tools (default 1).
```

**Comment by RamonUnch** [comment]
Sorry the above is not the good one, maybe try with this one:
[hooks.zip](https://github.com/user-attachments/files/17925584/hooks.zip)

**Comment by RamonUnch** [comment]
you use x64 version?

Try with the combo altsnap.exe hooks.dll
[AltSnap_test_x64.zip](https://github.com/user-attachments/files/17929809/AltSnap_test_x64.zip)
[AltSnap_test_i386.zip](https://github.com/user-attachments/files/17929815/AltSnap_test_i386.zip)

**Comment by emvaized** [comment]
Yes, I'm using x64 version. 
Just tried the "test_x64" file, but still the same... Background service doesn't start

**Comment by RamonUnch** [comment]
be sure you look at the properties of the exe/dll file to check that they are not blocked for execution, this will happen on Win10/11 if you use the native zip extraction method.

**Comment by emvaized** [comment]
> check that they are not blocked for execution

How could I check it? There is no "read only" checkbox in the file properties. I also just tried to extract the archive using NanaZip, still same result

**Comment by RamonUnch** [comment]
https://www.tenforums.com/tutorials/5357-unblock-file-windows-10-a.html


I do not get it otherwise
Otherwise try with the 32bit version?

**Comment by emvaized** [comment]
> https://www.tenforums.com/tutorials/5357-unblock-file-windows-10-a.html
> 
> I do not get it otherwise Otherwise try with the 32bit version?

It's my bad, I did not notice that you also attached modified .exe file in the x64 archive. 

AltSnap now indeed only sends window move event only when window move was actually started — so it behaves much more consitently. However, mouse cursor still changes to the "move" one once I click on window, but didn't move it yet (I use `UseCursor=4` config). 

I understand that changing a cursor is a way of providing visual feedback to the user that the window will be moved once they move their cursor — so I'm not sure about if this should be changed. My use case for this is probably too much specific
